deps/chakrashim/src/inspector/v8-debugger.cc

```diff
@@ -77,22 +77,25 @@ void V8Debugger::getCompiledScripts(
     int contextGroupId,
     std::vector<std::unique_ptr<V8DebuggerScript>>& result) {
   JsValueRef scripts;
-  JsDiagGetScripts(&scripts);
+  if (JsDiagGetScripts(&scripts) != JsNoError) {
+    assert(false);
```

Comment:
Why is this assert needed? Seems like this is a legitimate situation that just needs to be handled?